katdal/chunkstore_s3.py

```diff
@@ -75,6 +76,7 @@ def __init__(self, client):
             raise _botocore_import_error
         error_map = {EndpointConnectionError: StoreUnavailable,
                      ConnectionError: StoreUnavailable,
+                     requests.exceptions.ReadTimeout: StoreUnavailable,
```

Comment:
Shouldn't this catch `requests.exceptions.RequestsException`? There are lots of exception cases other than ReadTimeout (http://docs.python-requests.org/en/master/_modules/requests/exceptions/)

Reply:
Thanks, I was looking for such a more general connection-related catch-all but looked on the wrong side (the botocore client side, that is). I'll use that.

Reply:
One thought: does error_map work with the class hierarchy or do you need an exact type match?

Reply:
Both of sorts, actually. It first looks up the exact type and if not found, looks for the first key in error_map that is a base class of the exception.